sig.js

```diff
@@ -1,6 +1,11 @@
 import { createHash, createCipheriv, hkdfSync, scryptSync, randomUUID, randomBytes } from 'node:crypto'
 
-export const randomSeed = () => randomBytes(32)
+export const randomSeed = () => {
+  console.debug("sig | randomSeed | Called.");
+  const seed = randomBytes(32);
+  console.debug(`sig | randomSeed | Returning seed ${seed}.`);
+  return seed;
```

Comment:
Similarly here maybe we could log something more readable... I guess `seed.toString("hex")`